libsigner/src/v0/messages.rs

```diff
@@ -1330,7 +1330,9 @@ impl StacksMessageCodec for RejectReason {
     }
 
     fn consensus_deserialize<R: Read>(fd: &mut R) -> Result<Self, CodecError> {
-        let type_prefix_byte = read_next::<u8, _>(fd)?;
+        let Ok(type_prefix_byte) = read_next::<u8, _>(fd) else {
+            return Ok(RejectReason::NotRejected);
```

Comment:
Should this map to `Unknown` instead of `NotRejected`?

Reply:
Maybe `Unknown(RejectReasonPrefix::Unknown as u8)`?

Reply:
Yep good call!